commands/firestore-indexes-list.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+var Command = require('../lib/command');
+var firestoreIndexes = require('../lib/firestore/indexes.js');
+var requireAccess = require('../lib/requireAccess');
+var scopes = require('../lib/scopes');
+var logger = require('../lib/logger');
+
+var _prettyPrint = function(indexes) {
+  indexes.forEach(function(index) {
+    logger.info(firestoreIndexes.toPrettyString(index));
+  });
+};
+
+var _makeJsonSpec = function(indexes) {
```

Comment:
Could do this whole thing with a [lodash pick](https://lodash.com/docs/4.17.4#pick):

```js
// make sure to require lodash above
return {
  indexes: indexes.map(function(index) {
    _.pick(index, ['collectionId', 'fields'])
  })
}
```

I'm assuming that `indexes` that gets passed in has extra fields you don't want? Otherwise, you don't even need to do that.